pyproject.toml

```diff
@@ -83,6 +84,7 @@ dev = [
     "tifffile",
     "tox-direct",
     "types-mock",
+    "tiled[all]",
```

Comment:
Not sure this is right; I think this might not play well with `pip install .[dev]` for example. I think our options are:

- Recommend `pip install ,[dev,all]` using `all` explicitly, or
- Copy all the requirements from `all` into `dev.

Reply:
Doing `tiled[all,dev]`